### PR TITLE
Add additional parameter to enable a different TTL for updates

### PR DESCRIPTION
**Pull request by EvgenIn**
This change will allow to set a different (shorter) TTL for updates. 
It will help in situations when a node crashes during this step and with the default TTL other nodes will not perform the repair for a long time. 
This is backward compatible and does not force the user to use this the parameter.

### CODE REVIEW

File: cassandra_repair_scheduler.py [hunk 1]
```diff
@@ -262,16 +263,19 @@ def run_repair(self):
                "--dry-run"]     # So we get a list of commands to run.
         if self.option_group.local:
             cmd.append("--local")
+	if self.option_group.parallel:
+	    cmd.append("-p")
         logging.debug("geting repair steps, this may take a while")
-        repair_steps = subprocess.check_output(cmd).split('\n')
+        repair_steps = subprocess.Popen(cmd,stdout=subprocess.PIPE,stderr=subprocess.STDOUT,).communicate()[0].split('\n')
```
Comment (BrianGallew) [inline cassandra_repair_scheduler.py]: This seems like a pointless change here, unless you're planning on doing something strange elsewhere.

File: cassandra_repair_scheduler.py [hunk 2]
```diff
@@ -493,13 +502,13 @@ def cli_parsing():
     parser.add_argument("--reset", action="store_true", default=False,
                         help="Reset the repair status for the host")
     options = parser.parse_args()
+    options.ttl_repair = args.ttl_repair if args.ttl_repair else args.ttl
```
Comment (BrianGallew) [inline cassandra_repair_scheduler.py]: where does args.ttl_repair come from?  Shouldn't that be options.ttl_repair?  Likewise args.ttl?

File: cassandra_repair_scheduler.py (hunk 1 above)
Comment (EvgenIn) [inline cassandra_repair_scheduler.py]: this is now removed from the PR

File: cassandra_repair_scheduler.py (hunk 2 above)
Comment (EvgenIn) [inline cassandra_repair_scheduler.py]: correct, I fixed this

File: cassandra_repair_scheduler.py (hunk 1 above)
Comment (imriz) [inline cassandra_repair_scheduler.py]: @BrianGallew this was done to support python <2.7

Comment (BrianGallew) [inline cassandra_repair_scheduler.py]: Thanks for the clarification.

**Pull request merged**